wire-ios/Wire-iOS/Sources/Authentication/Coordinator/AuthenticationCoordinator.swift

```diff
@@ -432,7 +432,7 @@ extension AuthenticationCoordinator {
 
     func makeUnregisteredUser() -> UnregisteredUser {
         let user = UnregisteredUser()
-        user.accentColor = .random
+        user.accentColor = .blue
```

Comment:
that's it !? 
what color does a returning user get ? is it another code path?

Reply:
As far as I understood, that's it! They get the blue color. I tested it by creating a new user 3 times. All of those 3 times they got the blue accent color. @netbe